openassessment/templates/openassessmentblock/staff_area/staff_area.html

```diff
@@ -0,0 +1,171 @@
+{% load i18n %}
+{% load tz %}
+<div id="openassessment__staff-area" class="wrapper--staff-area">
```

Comment:
As we discussed, this is a known issue with ORA that I haven't made worse:

https://openedx.atlassian.net/browse/TNL-1593

I've marked this bug as ready for grooming so we can size it and decide when/if to tackle this.